CHANGELOG.rst

```diff
@@ -1,6 +1,29 @@
 Change log
 ==========
 
+0.19.1 (2020-11-06)
+-------------------
+
+New features
+~~~~~~~~~~~~
+
+* **config:** add :class:`~universum.configuration_support.Step` and :class:`~universum.configuration_support.Configuration`
+  classes to :doc:`configuration support <configuration_support>` for type checking and increased usability
+* **config:** ``-cfg``/``--config`` :doc:`command line argument <args>` is no longer mandatory; if not specified,
+  the default value (``.univerum.py`` in project root) is used
+* **config:** add ``universum init`` :doc:`command <additional_commands>` for automatic configuration file creation
+* **add ``universum run`` :doc:`command <additional_commands>` to launch Universum in Non-CI mode**
```

Comment:
It looks like entire string made bold messes up the sphinx rendering, see https://universum.readthedocs.io/en/update-changelog/changelog_ref.html